packages/measured-reporting/test/unit/registries/test-SelfReportingMetricsRegistry.js

```diff
@@ -123,13 +123,29 @@ describe('SelfReportingMetricsRegistry', () => {
   it('#getOrCreateCachedGauge creates and registers the metric and when called a second time returns the same metric', () => {
     mockReporter.expects('reportMetricOnInterval').once();
 
-    const metric = selfReportingRegistry.getOrCreateCachedGauge('the-metric-name', () => {
-      return new Promise((r) => { r(10); });
-    }, 1, {}, 1);
-
-    const theSameMetric = selfReportingRegistry.getOrCreateCachedGauge('the-metric-name', () => {
-      return new Promise((r) => { r(10); });
-    }, 1, {}, 1);
+    const metric = selfReportingRegistry.getOrCreateCachedGauge(
```

Comment:
Formatting a bit rough, not sure if yarn format can ignore?  Or maybe it is fine.